README.md

```diff
@@ -6,7 +6,69 @@
   <a href="https://circleci.com/gh/kids-first/kf-ui-data-tracker"><img src="https://img.shields.io/circleci/project/github/kids-first/kf-ui-data-tracker.svg?style=for-the-badge"></a>
 </p>
 
-Kids First Data Tracker
-=======================
+# Kids First Data Tracker
 
 Investigator study management and progress tracking.
+
+## Development Quick Start
+
```

Comment:
Might be nice to mention the dependency on the api here, as this codebase isn't really functional without it.

Reply:
Added "Study-creator served as backend provides the data APIs consumed by data-tracker frontend."